newuser/newuser.js

```diff
@@ -0,0 +1,292 @@
mQDJQAJE3TimUv71170a+tc4BnAcRK2S/1SOqU9F3rLEgGnh2ApKhvbd3UtHzZUoZ/fLACEFOO671N0vescyAZWAlAnDUuWfzNwc1bzhredbV1FExCAYix6q2C90kqWGdB/FECEEdNy5e9rrfnop9b58DTcQogxsp+6UFJbZKusM6C+ON0AOKkvq11XXH50qUM//hiBSAhHNc7XdI11jkQf6wEwFpxyWJvz+YtH2HJP94oAAniuN5KVbYQ5oFCmBYlABby9fXDjZ0dp/T2XHSrdRbMjI2AEqSvx+9r7eq+XNIOSS+3zoP4YrMgRK2xY9F/1hUbD+/bsGqjdRbMDisACeW43qGS/iip2ToL4ouVAEShuGzpW/rWXfxj6xyYG1YAEqqvx9/c2tX9FVUu5DzZOg/iiZUAhKmhfeGf6tvaDu1bf8kd1lkwd6wApIDjesdKut06B+KLlQAErbhs6T/1rbuYO5QSjBWAFOjr8cutXd3nq/L/54us8yB+WAlAUBraF95V39Z2WN/6S262zoL5YQUgZRzXO1LSLeLaAEyBlQDMR3HZknf3rbvkW9Y5EAxWAFKmr8d/vLWr+8uS+iWdZp0H8cJKAGrRsKj9xkLLgiP6N6y6zToLgsMKQIpVny74M7FvAPbBSgBmI1cojDR2dry2b93Fv7TOguCxFXCKVZ8ueJSkt1hnQbywbTBm0ri488q6YrGJ4Z9erABkhON6iySdL+ls6yyID1YCsK9CS8vGQnPTy/vWX/KgdRaEiwKQMY7rHSXpV5KWW2dBPFACMKG4dMk/9q2/5CrrHIgGBSCDHNfLSzpT0vessyAeKAHZ1tjZ8b3x0dF/2vXolXussyA6FIAMc1xvoaRPSSpZZ4E9SkD21Le1PlBXLJ7Rt/4S/o/PIAoAJu4WuFLSi62zwBYlIBvyDQ27G9oXvrlv/SXXWWeBHQoAnuK43omSfiGp3ToL7FAC0q1xyeKPD27e8uXhXVePW2eBLQoA9lK9PuDvJF1tnQV2KAHp09jZccXYyMg5A+WrdltnQTxQADAlx/WKks6SdJF1FtigBKRDw6L2/8jl8+/s33TZk9ZZEC8UAEzLcb02SedIOs86C6JHCUiu+oVtt+cbGs7s37BqrXUWxBMFALPiuF6npI9V/0KGUAKSpb619S/5YuOZ/RtW3WOdBfFGAcCcOK63VNInJH3YOguiQwmIv0Jry5pCsfi2vg2r/midBclAAUBNHNdbJumjYg+BzKAExFN9a8tD+WLxHTypD3NFAcC8OK63WNIHVdlQCClHCYiP+rbWe/KNje/u37DqTussSCYKAAJR3VXwLEmedRaEixJgq37hwpvy9YX39G+8lIf1YF4oAAiU43pNkt4g6bvWWRAeSkD0Gha1/ySXz3+4f9NlZessSAcKAELhuF6dpJdKWiWpyzoPgkcJiEZjZ8eXxkZGPjdQvqrfOgvShQKAUDmuJ0mHSTpXlZUBpAglIBx1xeKOQsuCfxnauu07Q/0/GbPOg3SiACAy1VsIz5L079ZZEBxKQHDq29puyzfUn9O/8VKu6EfoKACInON6DZJOVeWCwUOs82D+KAHz09jZ8fXxsbHP73rkii3WWZAdFACYclzvIEnvE/sJJB4lYG4KLS1r6oqNHx3csvVanswHCxQAxEL14UMvk/R5SUdY50FtKAEzyOXGGxe1XzQufWHXw5c/Zh0H2UYBQOw4rrdS0lslnW+dBXNHCXi6+oVtt+br6/9tcMvW3/GnfcQFBQCx5bheXpXVgH+WdLZ1HsweJUAqNDeX65qbPje6Z/C7u5/41oB1HmBfFAAkQvUUwQtUeQjRa6zzYGZZLAH5xob++paWr4+PjV+865HLH7fOA0yHAoDEcVyvRdLJkj4k6TTrPNi/LJSAfH39nvq21kslXdi/6bJ11nmA2aIAINGqZeBESe8XKwOxlMYSkG9s7K9vWXCZpEsY+kgqCgBSo/ocgqMlvUNcMxAraSgBhQXNj9Y1NV2ssbHv9D98OfvxI/EoAEil6rMIni3pdEkfkbTEOlPWJbEE1C9suyXf0HDx2NDQdezFj7ShACATHNfrlPR8SW+X9GbrPFkV9xJQ19S0tdDc9H3l898e3LzlHm7ZQ5pRAJA51dWBAyW9RNI7JZ1knSlL4lQC8vX1Q4WWBb/MFwrfGh0aumGgfNUu60xAVCgAyLzqswkOknSKKhsQvcg6U9pZlYB8ff1goWXB9flC4fvjo6PX9z98+TbrzwKwQgEA9uG4XkHSMyQdr8o1BGdaZ0qjKEpAobn5ibqm4i9zdXU/HRsZuWXXw5f3Wh83EBcUAGAGjutJ0iJJz1HllsMzVNmHAPMUZAnINzbsLjQ335qvL1wj5fzhvr41e7Z8b9T6GIG4ogAANahuU9wh6VmSjlNlQyL2IahBLSWgrrlpe12xeGu+UPi1crkbxwYHH9r16JWD1scCJAkFAAhIdaWgWdIKSQdLOkaVlYJTrbPF3f5KQF1TU19dsfHefH3hJuXz/6Vx3T28c+cTe7Z+f8w6M5B0FAAgZNViUFRlxeAAVU4lHKbKNQYnSSpYZzQ0IulmSbf3b9hYGB8dfSiXy909Pj6+ZnhH7/Y9W7/HbXhASCgAgLHqRYcLVLnOYLkqJeGZqmxkdIikQ1UpD0mzTdL9kh6UtEbSekmPSHpc0nZJu8p+acQ6JJBV/x9iXVBA6uYbKQAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0wMy0wN1QwNjo0MzowOCswMDowMI3/JdUAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMDMtMDdUMDY6NDM6MDgrMDA6MDD8op1pAAAAY3RFWHRzdmc6Y29tbWVudAAgR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAxOS4wLjAsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICDOSJALAAAAAElFTkSuQmCC')
```

Comment:
Can't we save the images as files and load them as normal images? The inline-data for images makes it harder to read the code

Reply:
updated as review comment